connectivity/http/http.c

```diff
@@ -9,12 +9,16 @@
 #include "utils/macros.h"
 
 #define HTTP_LOGGER "http"
+#define MAX_REQUEST_LEN 65536
 
 static logger_id_t logger_id;
 
 typedef struct ta_http_request_s {
   bool valid_content_type;
+  char *answer_string;
+  uint32_t answer_code;
```

Comment:
I am confused with answer code. Is it related to status code?
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
Yes, it is the status codes defined [here](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).